server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -338,9 +338,10 @@ private void write(final Collection<CommitSession> sessions, boolean mincFinish,
           success = (currentLogSet == logSetId.get());
         }
       } catch (DfsLogger.LogClosedException ex) {
-        log.debug("Logs closed while writing, retrying attempt " + writeRetry.retriesCompleted());
+        writeRetry.logRetry("Logs closed while writing");
```

Comment:
When I saw this code is made me think about also logging when the message is different.

Reply:
yea, I thought about that but also thought that perhaps that is going to far.